### PR TITLE
Remove news card feature

### LINKED ISSUE #14204: Remove NewsCard
As we have the new feature announcement and the new designs for Reader, please remove the News card feature

### PR DESCRIPTION
**Pull request by negna1**
Fixes #14204

To test: 
1. log in wordpress
2. Tap to reader tab bar
3. there must be any header which tells 

PR submission checklist:

- [X] I have considered adding unit tests where possible.
- [X] I have considered adding accessibility improvements for my changes.
- [X] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Review by frosty — CHANGES_REQUESTED** [review]
Thanks for working on this issue!

Please can you ensure that your branch is up to date with the latest code from `develop`, and only contains commits relevant to the changes here? As well as a lot of merge commits, I also see this one: `remove empty box flash on menu item`, which I think is from one of your other PRs.

Also, please could you remove the code you've commented out completely? We generally don't like to leave commented out code in the project, and it can always be restored from git if we needed it in the future. I noticed you removed a number of files from the Xcode project but left the actual files present on disk. We can also remove the `News.strings` file, which contains the actual news items.

File: WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift [hunk 1]
```diff
@@ -483,56 +483,19 @@ import WordPressFlux
 
     // MARK: - Configuration / Topic Presentation
 
-    @objc private func configureStreamHeader() {
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift:486]: This method shouldn't be removed, as it is required to display the header view for many parts of the Reader.

File: WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift [hunk 2]
```diff
@@ -483,56 +483,19 @@ import WordPressFlux
...
     // topic's following status changed.
+    
+    //remove news card . as I see ther is no need to be thisfunction because there must not be any header , but it is possible to add some
+    //header , so i only delete configureStreamHeader()
     private func refreshTableHeaderIfNeeded() {
         guard let _ = readerTopic else {
             return
         }
-        configureStreamHeader()
+       // configureStreamHeader()
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift:495]: We still need to call this method here to provide a header to various views in the Reader.

File: WordPress/Classes/ViewRelated/System/WPTabBarController.m [hunk 3]
```diff
@@ -120,16 +120,8 @@ - (instancetype)init
                                                  selector:@selector(updateIconIndicators:)
                                                      name:NSNotification.ZendeskPushNotificationClearedNotification
                                                    object:nil];
+//remove news card notifications
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/System/WPTabBarController.m:123]: There's no need to include a comment here :)

**Review by frosty — CHANGES_REQUESTED** [review]

File: WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift (hunk 1 above)
Comment (negna1) [reply WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift:486]: Okay , I see. Nothing was using this method and that's why i deleted it. I will try to fix what you  say.

**Comment by twstokes** [comment]
👋 I'm going to close this PR due to lack of activity, but please reopen if this was an error. Thank you.

**Pull request closed without merging**